app/controllers/merchant_items_controller.rb

```diff
@@ -1,3 +1,6 @@
 class MerchantItemsController < ApplicationController
-  def index; end
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
+    @items = @merchant.items
```

Comment:
is this restful?

Reply:
This is fine, but it would be better to delete line 4 and put this ".items" action in the view.

Reply:
This isn't really a case of being "RESTful" or not, this is a "Skinny Controller" thing.